plugins/source/airtable/docs/_authentication.md

```diff
@@ -0,0 +1 @@
+In order to fetch information from Airtable, `cloudquery` needs to authenticate using a [personal access token for Airtable's API](https://airtable.com/developers/web/guides/personal-access-tokens). Follow the instructions on the Airtable website and create a read-only token with `data.records:read` and `schema.bases:read` scopes.
```

Comment:
Docs need to be under the plugin now instead of under Website. I'll remove the Website docs after we have the plugin available in the Hub